docs/router/README.md

```diff
@@ -0,0 +1,9 @@
+# The router book
+
+## How to run the tests
+- Navigate your terminal to this directory
+- Run `cargo clean`
+- Run `cargo build --features router,ssr,web`
+- Run `mdbook test -L ../../target/debug/deps/`
+
+I don't know if there is a better way, but it works for me.
```

Comment:
That file isn't part of the rendered mdbook, so not sure if its part of the official docs. Also, if someone knows a better way to do this, I'd be happy to switch to it.

Reply:
I don't think it's necessarily better, but an alternative would be organizing it the same way the Dioxus Guide is organized. I basically made it a crate (with an empty lib.rs), put all the snippets in `examples` (so that they can be run/tested like regular cargo examples) and included as [MdBook snippets](https://rust-lang.github.io/mdBook/format/mdbook.html#including-portions-of-a-file). Tbf, my `#include` directives kinda looked ugly due to the distant relative path.

But I think the current approach is good. I would just suggest removing this line, cos it looks a bit unprofessional (even if it's not in the book, it will still show up on GitHub). Yeah, the commands do look a bit more hacky than usual, but if it works, it works.

Reply:
Oh, you just wanted me to remove the line :sweat_smile: then I'll do just that :+1: